src/libsyntax/edition.rs

```diff
@@ -55,6 +55,13 @@ impl Edition {
             Edition::Edition2018 => "edition_2018",
         }
     }
+
+    pub fn feature_name(&self) -> &'static str {
+        match *self {
+            Edition::Edition2015 => "rust_2015_preview",
```

Comment:
Avoiding the anachronism of "preview" may be more valuable than keeping these names symmetrical?

Reply:
No, once a feature is added it can't be "removed" completely.

There's no way to use the rust_2015_preview feature, that feature exists only so that attempting to use it will throw a "feature removed" warning. This goes for the 2018_preview feature when you enable the 2018 epoch, as well.

Yes, we could just not do this for 2015, but it didn't seem worth it to special case 2015.